R/report_dispatcher.R

```diff
@@ -13,15 +13,25 @@
 #' \code{studentids, depth_string} into the arg list, as well as named elements 
 #' corresponding to the key/value cut and element outlined above
 #' @param calling_env defaults to parent frame.
+#' @param post_process a post processing function to apply to the list of plots we get back.
+#' default behavior is only_valid_plots(), which drops any plot that failed.  
+#' don't want that?  write something new :)
+#' @param verbose should the function print updates about what is happening?  default is TRUE.
 #' @param ... other parameters to pass through (namely cdfs).  todo: reformat this function
 #' to take mapvizieR object.
 #' 
 #' @export
 #' 
-#' @return a cdf with new term fields 
+#' @return a list of output from the function you called
 
-report_dispatcher <- function(mapvizieR_obj, cut_list, call_list, 
-    func_to_call, arg_list=list(), calling_env = parent.frame(), ...
+report_dispatcher <- function(
+    mapvizieR_obj,
+    cut_list, call_list, 
+    func_to_call, arg_list=list(),
+    calling_env = parent.frame(),
+    post_process = "only_valid_plots",
```

Comment:
You can all pass this as 
```R
post_process =  only_valid_plots
```
i.e., without the quotes. (I think, since do call will look up the function name either way.)